packages/terra-collapsible-menu-view/src/CollapsibleMenuViewDivider.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Menu from 'terra-menu';
+import 'terra-base/lib/baseStyles';
+import styles from './CollapsibleMenuView.scss';
+
+const cx = classNames.bind(styles);
+
+const contextTypes = {
+  isCollapsibleMenuItem: PropTypes.bool,
+};
+
+const CollapsibleMenuViewDivider = (props, { isCollapsibleMenuItem }) => {
```

Comment:
Is props here intentional?

Reply:
I needed to accept props in order to accept the context.